build-toolchain.bash

```diff
@@ -155,7 +160,7 @@ if [ $SKIP_THIRDPARTY != false ]; then
 		if [ ! -d binutils-build-ppc ]; then MISSING=true; fi
 		if [ ! -d gcc-build-ppc ]; then MISSING=true; fi
 	fi
-	if [ ! -d hfsutils ]; then MISSING=true; fi
+	if [ $SKIP_HFSUTILS != false -a ! -d hfsutils ]; then MISSING=true; fi
```

Comment:
In the following check where "Not all third-party components have been built yet, ignoring --skip-thirdparty" is printed, you also need to set `SKIP_HFSUTILS=false`.

Reply:
And isn't this condition backwards as well? Shouldn't it be:
```suggestion
	if [ $SKIP_HFSUTILS = false -a ! -d hfsutils ]; then MISSING=true; fi
```